freud/order.pyx

```diff
@@ -740,6 +740,31 @@ cdef class LocalQl(Compute):
     def __str__(self):
         return repr(self)
 
+    @Compute._computed_method()
+    def plot(self, ax=None):
+        """Plot Ql.
+
+        Args:
+            ax (:class:`matplotlib.axes`): Axis to plot on. If :code:`None`,
+                make a new figure and axis. (Default value = :code:`None`)
+
+        Returns:
+            (:class:`matplotlib.axes`): Axis with the plot.
+        """
+        from plot import histogram_plot as hplot
```

Comment:
Let's skip the abbreviation here. It's only used once and it's not a standard abbreviation like `np` for `numpy`. One reason that we try to avoid this is that it's hard to search the codebase for specific functions if they're abbreviated in unexpected ways. If you're having difficulty with long line lengths, just wrap to a new line after `return histogram_plot(`.